### PR TITLE
Report more errors to sentry

### PR DESCRIPTION
**Pull request by piuccio**
Make sure async requests failures are reported to sentry

### CODE REVIEW

File: facia-tool/public/js/models/collections/collection.js [hunk 1]
```diff
@@ -292,7 +298,12 @@ define([
                 self.state.timeAgo(self.getTimeAgo(raw.lastUpdated));
             });
         })
-        .catch(function () {})
+        .catch(function (ex) {
+            // Netowrk errors should be ignored
```
Comment (uplne) [inline facia-tool/public/js/models/collections/collection.js]: Typo

File: facia-tool/public/js/models/collections/collection.js [hunk 2]
```diff
@@ -292,7 +298,12 @@ define([
...
+            if (!_.isNumber(ex.status)) {
```
Comment (uplne) [inline facia-tool/public/js/models/collections/collection.js]: What about isInteger

**Comment by uplne** [comment]
Other than that :+1:

**Comment by piuccio** [comment]
I messed up the branch pushing another commit, so I amended. The new commit fixed the typo and uses `instanceof` instead of `Number`.

File: facia-tool/public/js/models/collections/article.js [hunk 3]
```diff
@@ -862,8 +865,9 @@ define([
 
                 self.updateEditorsDisplay();
             })
-            .catch(function() {
+            .catch(function(ex) {
                 self.meta.headline(undefined);
+                reportErrors(ex);
```
Comment (OliverJAsh) [inline facia-tool/public/js/models/collections/article.js]: BTW, `reportErrors` will rethrow the error unless you tell it not to: `reportErrors(error[, options, shouldRethrow])`

Comment (piuccio) [inline facia-tool/public/js/models/collections/article.js]: I'm not sharing frontend reportErrors, facia-tool has its own version which is rather dumb.

Comment (OliverJAsh) [inline facia-tool/public/js/models/collections/article.js]: Ah, cool!

On Thu, 27 Aug 2015 at 10:24 Fabio Crisci <notifications@github.com> wrote:

> In facia-tool/public/js/models/collections/article.js
> <https://github.com/guardian/frontend/pull/10229#discussion_r38077105>:
>
> >                  self.meta.headline(undefined);
> > +                reportErrors(ex);
>
> I'm not sharing frontend reportErrors, facia-tool has its own version
> which is rather dumb.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/10229/files#r38077105>.
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Pull request merged**